src/htm/engine/RESTapi.cpp

```diff
@@ -36,44 +36,43 @@ RESTapi::~RESTapi() { }
 
 RESTapi* RESTapi::getInstance() { return &rest; }
 
-std::string RESTapi::get_new_id_(const std::string &old_id) {
-  std::string id = old_id;
-
-  // re-use the same id if we can or use a provide one.
+  std::string RESTapi::get_new_id_() {
+  // No id was provided so find the next available number.
+  // Do not use an id of "0".
```

Comment:
if ID `0` has a special meaning, is there an assert so the user cannot set it?

Reply:
Oh, I intended to change that comment.   I will clarify.